crates/parcel_core/src/types/asset.rs

```diff
@@ -76,12 +101,13 @@ pub struct Asset {
   /// This can be used to find assets during packaging, or to create dependencies between multiple
   /// assets returned by a transformer by using the unique key as the dependency specifier.
   ///
+  /// TODO: Make this non-nullable and disallow creating assets without it.
```

Comment:
If we drop asset groups (which I think we should) then we might be able to remove this entirely?

Reply:
We should remove this